templates/taxi/car_list.html

```diff
@@ -1,17 +1,12 @@
 {% extends "base.html" %}
+{% load crispy_forms_filters %}
 
 {% block content %}
-  <h1>Car list</h1>
-  {% if car_list %}
-    <ul>
-      {% for car in car_list %}
-        <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
-          {{ car.model }} ({{ car.manufacturer.name }})
-        </li>
-      {% endfor %}
-    </ul>
-  {% else %}
-    <p>There are no cars in taxi</p>
-  {% endif %}
+  <h1>{{ object|yesno:"Update, Create" }} Car </h1>
```

Comment:
The use of the `yesno` filter with `object` is incorrect here. The `yesno` filter is typically used with boolean values to display different text based on the value. Ensure that `object` is a boolean or adjust the logic to correctly display 'Update' or 'Create' based on the context.